code/datums/wounds/burns.dm

```diff
@@ -164,6 +164,8 @@
 	. = ..()
 	. += "<div class='ml-3'>"
 
+	. += "Flesh Damage - Healing: [flesh_damage] - [flesh_healing]\n"
```

Comment:
What kind of units is this actually using?

Reply:
Uhh its just display the amounts of damage

Reply:
I feel like it could be displayed a bit better, maybe show it as estimated time to recovery instead?

Reply:
Ur right it's a bit iffy